squad/core/management/commands/update_project_statuses.py

```diff
@@ -0,0 +1,30 @@
+from datetime import timedelta
+from dateutil import parser as date_parser
+from django.core.management.base import BaseCommand
+from django.utils import timezone
+
+from squad.core.models import ProjectStatus, Build
+
+
+class Command(BaseCommand):
+
+    help = """Update project status records in order to fix a bug which compared current builds with the wrong ones."""
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--date-start',
+            dest="date_start",
+            default=(timezone.now() - timedelta(days=180)),
+            type=date_parser.isoparse,
+            help="Start date for project status updates (default: 6 months before current date, format: YYYY-MM-DD)."
+        )
+
+    def handle(self, *args, **options):
+        self.options = options
+
+        builds = Build.objects.filter(
+            datetime__gt=timezone.make_aware(self.options['date_start']),
```

Comment:
I guess date_start should be checked for existence. Alternatively we can set some default value (beginning of epoch?) to update all ProjectStatus objects. Without the parameter I'm getting this:
```
Traceback (most recent call last):
  File "./manage.py", line 5, in <module>
    main()
  File "/home/milosz/linaro/tcstorm/squad/squad/manage.py", line 14, in main
    execute_from_command_line(sys.argv)
  File "/home/milosz/linaro/tcstorm/squad_venv/lib/python3.5/site-packages/django/core/management/__init__.py", line 364, in execute_from_command_line
    utility.execute()
  File "/home/milosz/linaro/tcstorm/squad_venv/lib/python3.5/site-packages/django/core/management/__init__.py", line 356, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/home/milosz/linaro/tcstorm/squad_venv/lib/python3.5/site-packages/django/core/management/base.py", line 283, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/home/milosz/linaro/tcstorm/squad_venv/lib/python3.5/site-packages/django/core/management/base.py", line 330, in execute
    output = self.handle(*args, **options)
  File "/home/milosz/linaro/tcstorm/squad/squad/core/management/commands/update_project_statuses.py", line 26, in handle
    datetime__gt=timezone.make_aware(self.options['date_start']),
  File "/home/milosz/linaro/tcstorm/squad_venv/lib/python3.5/site-packages/django/utils/timezone.py", line 285, in make_aware
    return timezone.localize(value, is_dst=is_dst)
  File "/home/milosz/linaro/tcstorm/squad_venv/lib/python3.5/site-packages/pytz/__init__.py", line 237, in localize
    raise ValueError('Not naive datetime (tzinfo is already set)')
ValueError: Not naive datetime (tzinfo is already set)
```